From 381eae22927c9ccceca5b0619f8ca761921f1c79 Mon Sep 17 00:00:00 2001
From: Marco Franceschetti <vonfritz1@gmail.com>
Date: Mon, 17 Mar 2014 19:49:47 +0100
Subject: [PATCH] enable VDPAU bitmap surface

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index cf76c0e..17fe288 100644
--- a/Makefile
+++ b/Makefile
@@ -41,7 +41,7 @@ CONFIG += -DUSE_PIP			# PIP support
 #CONFIG += -DUSE_MPEG_COMPLETE		# support only complete mpeg packets
 #CONFIG += -DH264_EOS_TRICKSPEED	# insert seq end packets for trickspeed
 #CONDIF += -DDUMP_TRICKSPEED		# dump trickspeed packets
-#CONFIG += -DUSE_BITMAP			# VDPAU, use bitmap surface for OSD
+CONFIG += -DUSE_BITMAP			# VDPAU, use bitmap surface for OSD
 CONFIG += -DUSE_VDR_SPU			# use VDR SPU decoder.
 #CONFIG += -DUSE_SOFTLIMIT		# (tobe removed) limit the buffer fill
 
-- 
1.8.2.2

